vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -155,13 +155,16 @@ ent prj install
 ent prj install --conflict-strategy=OVERRIDE
 ```
 
+  > **Note**
+  >
+  > In the event of a timeout or pod failure, refer to the errors captured by the quickstart-cm-deployment and plugin deployer logs. 
+
 ## Use the App Builder
-1. Log into the `App Builder`
+1. Log into your App Builder and select `Component Repository` in the upper right
```

Comment:
Is this correct?

Reply:
Good catch, it's in the lower left.

Reply:
And the label is 'Repository'

Reply:
now you both know i don't always have the app open while i edit... caught red handed!